core/src/main/java/com/gentics/mesh/cli/CoreVerticleLoader.java

```diff
@@ -81,91 +66,45 @@ public void loadVerticles(List<String> initialProjects) {
-			.ifPresent(id -> searchVerticleId = id);
-	}
-
-	public Optional<String> loadVerticle(AbstractVerticle verticle) {
-		if (verticle == null) {
-			return Optional.empty();
-		}
-		try {
-			if (log.isInfoEnabled()) {
-				log.info("Loading mandatory verticle {" + verticle.getClass().getName() + "}.");
-			}
-			return Optional.of(deployAndWait(rxVertx.getDelegate(), defaultConfig, verticle, true));
-		} catch (Exception e) {
-			log.error("Could not load mandatory verticle {" + verticle.getClass().getSimpleName() + "}.", e);
-			return Optional.empty();
-		}
-	}
-
-	public Completable unloadVerticles() {
-		return Observable.merge(
-			searchVerticleId != null ? Observable.just(searchVerticleId) : Observable.empty(),
-			Observable.fromIterable(deploymentIds)).flatMapCompletable(rxVertx::rxUndeploy)
-			.doOnComplete(deploymentIds::clear);
+		deployRestVerticle();
+		deployMonitoringVerticle();
+		deployJobWorkerVerticle();
+		deploySearchVerticle();
 	}
 
-	public void reloadSearchVerticle() {
-		if (searchVerticleId != null) {
-			rxVertx.rxUndeploy(searchVerticleId).blockingAwait();
-			loadVerticle(resetSearchVerticle())
-				.ifPresent(id -> searchVerticleId = id);
-		}
+	private void deployRestVerticle() {
+		rxVertx.deployVerticle(restVerticle::get, new DeploymentOptions()
+			.setConfig(defaultConfig)
+			.setInstances(meshOptions.getHttpServerOptions().getVerticleAmount()));
 	}
 
-	/**
-	 * Return a Map of mandatory verticles.
-	 * 
-	 * @return
-	 */
-	private List<Provider<? extends AbstractVerticle>> getMandatoryVerticleClasses() {
-		List<Provider<? extends AbstractVerticle>> verticles = new ArrayList<>();
-		verticles.add(restVerticle);
+	private void deployMonitoringVerticle() {
 		if (meshOptions.getMonitoringOptions() != null && meshOptions.getMonitoringOptions().isEnabled()) {
-			verticles.add(monitoringServerVerticle);
+			rxVertx.deployVerticle(monitoringServerVerticle::get, new DeploymentOptions()
+				.setInstances(1));
 		}
-		return verticles;
 	}
 
-	/**
-	 * Get the map of mandatory worker verticle classes.
-	 * 
-	 * @return
-	 */
-	private List<AbstractVerticle> getMandatoryWorkerVerticleClasses() {
-		List<AbstractVerticle> verticles = new ArrayList<>();
-		verticles.add(jobWorkerVerticle);
-		return verticles;
+	private void deployJobWorkerVerticle() {
+		rxVertx.deployVerticle(jobWorkerVerticle, new DeploymentOptions()
+			.setInstances(1)
+			.setWorker(true));
 	}
 
-	private ElasticsearchProcessVerticle resetSearchVerticle() {
+	private void deploySearchVerticle() {
 		// Only deploy search sync verticle if we actually have a configured ES
 		ElasticSearchOptions searchOptions = meshOptions.getSearchOptions();
 		if (searchOptions != null && searchOptions.getUrl() != null) {
-			elasticsearchProcessVerticle = elasticsearchProcessVerticleProvider.get();
-			return elasticsearchProcessVerticle;
+			rxVertx.rxDeployVerticle(elasticsearchProcessVerticle, new DeploymentOptions()
+				.setInstances(1))
```

Comment:
Why only one instance?

Reply:
Before that change, there was also only one instance deployed.